src/ansys_sphinx_theme/announcement.html

```diff
@@ -0,0 +1,16 @@
+{% if theme_switcher %} 
+<div class="container-xl">
+    <div class="row"></div>
+        <div class="col-12 col-md-10" id="header-announcement">
+        <div class="col-12 col-md-13 col-xl-7" id="header-announcement-spacer">
+            <p id="announce"></p>
+            <script>
+            let file = "announcement.html"
```

Comment:
I was just thinking about the file type for the announcement. Do you think we should read from an `announcement.txt` file? That way we allow the theme to take the control of the whole CSS.

Reply:
Also, notice that this file is named `announcement.html` and this is reading from `announcement.html`, which causes an error. Could we rename this file like `announcement_layout.html`? This should be applied in the `layout.html` too.

Reply:
Both should work, txt or html format will work, anyway we are just replacing the content. so doesn't matter, if it is html, dont add the <p> first but we can add the href, bold etc in html file, which is an advantage.

Reply:
But thought of going with txt, because the developers can make custom announcement by just editing the txt file easily.